### PR TITLE
Missing translations for Chinese (zh_TW) #38733

### LINKED ISSUE #38733: Missing translations for Chinese (zh_TW)
Hello,

There are some missing translation for Chinese (zh_TW). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.zh_TW.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.zh_TW.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by fd6130**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| Tickets       | Fix #38733 
| License       | MIT

Added missing translations for zh_TW in Form, Security, Validator.

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                <source>Please enter an integer.</source>
+                <target>請輸入整數。</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>請選擇有效的語言。</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>請選擇有效的地區。</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf]: Same here, does it say "locale"?

File: src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>請輸入正確的金額。</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf:79]: This google translates to "please enter the **correct** amount", it should be "valid". Can you double check if this is correct? (Or if Google is wrong)

**Review by Nyholm — COMMENTED** [review]
Cool. 
I appreciate your work. 

Thank you. 

I just have some comments.

File: src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf (hunk 2 above)
Comment (fd6130) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf:79]: Hi, i'm not so sure about this cause the chinese word "正确" can be seen as "correct" or "valid" in english.  But to me, this is not wrong and  the google translates is not wrong as well.

File: src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf (hunk 1 above)
Comment (fd6130) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf]: Hi，i will change it to "請選擇有效的语言环境" 。

File: src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf (hunk 2 above)
Comment (fd6130) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf:79]: Or i stick to google translates, i change it to "有效" instead of "正确"

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf:79]: Do what sounds best. 

I just wanted to make sure you didn't accidentally made a mistake.

Comment (fd6130) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf:79]: Alright, so far so good.

**Review by Nyholm — APPROVED** [review]
Thank you so much!

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fd6130.

**Pull request merged**